src/AppInstallerCommonCore/Public/winget/Reboot.h

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+
+namespace AppInstaller::Reboot
+{
+    bool HasRebootPrivilege();
+
+    BOOL InitiateReboot();
```

Comment:
```suggestion
    bool InitiateReboot();
```
Avoid using non-standard types if possible.